src/Workspaces/Core/Portable/Options/IGlobalOptionService.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    /// <summary>
+    /// Provides services for reading and writing options.
+    /// This will provide support for options at the global level (i.e. shared among
+    /// all workspaces/services).
+    /// 
+    /// In general you should not import this type directly, and should instead get an
+    /// <see cref="IOptionService"/> from <see cref="Workspace.Services"/>
```

Comment:
I suspect the advice is more nuanced. If you're reading settings, you should not use this. If you're providing settings (i.e. syncing with something), you probably do want this.